src/core/server/base.js

```diff
@@ -125,6 +125,7 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
           includeSri: isDeployed,
           sriData,
           store,
+          trackingEnabled: config.get('trackingEnabled'),
```

Comment:
I was thinking it's weird that we include the GA script if we don't use it since tracking is enabled but there is no ID. I guess this shouldn't happen though so it should be fine.